mkt/lookup/views.py

```diff
@@ -329,7 +323,23 @@ def website_summary(request, addon_id):
     })
 
 
-@login_required
+@permission_required([('WebsiteLookup', 'View')])
+def website_edit(request, addon_id):
```

Comment:
heh, yeah. I guess I could change it. But everything else passes around `addon_id`s.